src/js/rx/containers/Active.jsx

```diff
@@ -22,6 +24,15 @@ class Active extends React.Component {
   constructor(props) {
     super(props);
     this.handleSort = this.handleSort.bind(this);
+
+    this.checkWindowSize = _.debounce(() => {
+      if (document.documentElement.clientWidth < 600) {
```

Comment:
How did you determine it was 600? Should it be assigned to a constant so it's clear what this limit is for?